raphtory-graphql/src/model/plugins/mutation_entry_point.rs

```diff
@@ -0,0 +1,26 @@
+use crate::model::algorithms::RegisterFunction;
+use async_graphql::dynamic::Object;
+use dynamic_graphql::internal::{OutputTypeName, Register, Registry, ResolveOwned, TypeName};
+use itertools::Itertools;
+use std::{collections::HashMap, sync::MutexGuard};
+
+pub trait MutationEntryPoint<'a>:
```

Comment:
This trait also seems to be exactly the same as the QueryEntryPoint. Maybe we can also have a PluginEntryPoint and implement it for both MutationPlugins and QueryPlugins